client/Elements/CreateTopic/CreateTopicWizard.patternfly.css

```
@@ -0,0 +1,41 @@
+.topics-wizard-content {
+  max-width: 500px;
+}
+
+.pf-c-form__group-control > * {
+  margin-bottom: 16px;
+}
+
+.form-in-wizard {
+  padding-top: 16px;
+}
+
+.radio-description {
+  padding-left: 23px;
```

23px is odd, suggest sticking to regulated sizes:
```suggestion
  padding-left: var(--pf-global--spacer--lg);
```

---

Done.